charts/zitadel/test/installation/config_test.go

```diff
@@ -58,3 +84,94 @@ func createSecret(ctx context.Context, namespace string, k8sClient *kubernetes.C
 	}, metav1.CreateOptions{})
 	return err
 }
+
+func testJWTProfileKey(audience, secretName, secretKey string) func(test *installation.ConfigurationTest) {
+	return func(cfg *installation.ConfigurationTest) {
+		t := cfg.T()
+		secret := k8s.GetSecret(t, cfg.KubeOptions, secretName)
+		key := secret.Data[secretKey]
+		if key == nil {
+			t.Fatalf("key %s in secret %s is nil", secretKey, secretName)
+		}
+		jwta, err := oidc.NewJWTProfileAssertionFromFileData(key, []string{audience})
+		if err != nil {
+			t.Fatal(err)
+		}
+		jwt, err := oidc.GenerateJWTProfileToken(jwta)
+		if err != nil {
+			t.Fatal(err)
+		}
+		closeTunnel := installation.ServiceTunnel(cfg)
+		defer closeTunnel()
+		var token string
+		installation.Await(cfg.Ctx, t, nil, 60, func(ctx context.Context) error {
+			var tokenErr error
+			token, tokenErr = getToken(ctx, t, audience, jwt)
+			return tokenErr
+		})
+		installation.Await(cfg.Ctx, t, nil, 60, func(ctx context.Context) error {
+			return callAuthenticatedEndpoint(ctx, audience, token)
+		})
+	}
+}
+
+func getToken(ctx context.Context, t *testing.T, audience, jwt string) (string, error) {
+	form := url.Values{}
+	form.Add("grant_type", "urn:ietf:params:oauth:grant-type:jwt-bearer")
+	form.Add("scope", "openid profile email urn:zitadel:iam:org:project:id:zitadel:aud")
```

Comment:
Could use the constants of the oidc library instead of the inline strings

Reply:
Using the oidc constants now :+1: